dotcom-rendering/src/web/components/SlotBodyEnd.importable.tsx

```diff
@@ -162,7 +193,11 @@ export const SlotBodyEnd = ({
 	}, [isSignedIn, countryCode, brazeMessages, asyncArticleCount]);
 
 	if (SelectedEpic) {
-		return <SelectedEpic />;
+		return (
+			<div css={wrapperStyle(contentType)}>
```

Comment:
This wrapper code can be replaced with `ContainerLayout`, which I suspect does the same thing but in a centralised manner so everything will line up nicely 📐

Reply:
I've just tried ContainerLayout but it's too wide -
![Screen Shot 2022-07-06 at 09 56 02](https://user-images.githubusercontent.com/1513454/177512291-866fce08-cee9-4cb0-8624-125cf29ba6ff.png)

I tried playing with some props but it doesn't change anything, e.g.
```
<ContainerLayout
	stretchRight={false}
	leftColSize={'compact'}
>
	<SelectedEpic />
</ContainerLayout>
```

Reply:
no effect

Reply:
🤔 I _feel_ like this should be possible but, equally, I might be wrong 🤷 

Could you ping me the steps required to test this? I'm happy to dig around a bit. I've got the branch running but I wasn't sure which url to use and how to ensure the slot shows? Also very happy to pair on this, if you like?